File: src/net/liplum/registries/CioItems.java
package net.liplum.registries;

import arc.graphics.Color;
import mindustry.ctype.ContentList;
import mindustry.type.Item;

public class CioItems implements ContentList {
    public static Item ic;

    @Override
    public void load() {
        ic = new Item("ic", Color.valueOf("272727")) {

        };
    }
}

File: src/net/liplum/registries/CioItems.kt
package net.liplum.registries

import mindustry.type.Item
import net.liplum.R

object CioItems : ContentTable {
    @JvmStatic lateinit var ic: Item
    override fun load() {
    }

    override fun firstLoad() {
        ic = Item(R.I.IC, R.C.IcDark)
    }

    override fun lastLoad() {
    }
}